charts/dawarich/templates/photon-loadbalancer/deployment.yaml

```diff
@@ -0,0 +1,77 @@
+{{- if .Values.photon.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "common.fullname" . }}-photon-loadbalancer
+  labels:
+    app.kubernetes.io/component: "photon-loadbalancer"
+    {{- include "common.labels" . | nindent 4 }}
+spec:
+  revisionHistoryLimit: 0
+  replicas: 1
```

Comment:
Since the proxy is stateless we can support HA configurations, replicas can be configurable.